Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -272,15 +290,19 @@ static OSStatus outputRenderCallBack(void *inRefCon,
 {
     OCTAudioEngine *myEngine = (__bridge OCTAudioEngine *)inRefCon;
 
-    int bytesToCopy = ioData->mBuffers[0].mDataByteSize;
+    UInt32 bytesToCopy = ioData->mBuffers[0].mDataByteSize;
     SInt16 *targetBuffer = (SInt16 *)ioData->mBuffers[0].mData;
 
     int32_t availableBytes;
-    SInt16 *buffer = TPCircularBufferTail(&myEngine->_buffer, &availableBytes);
-    int32_t sampleCount = MIN(bytesToCopy, availableBytes);
+    SInt16 *buffer = TPCircularBufferTail(&myEngine->_incomingBuffer, &availableBytes);
+    UInt32 sampleCount = MIN(bytesToCopy, availableBytes);
```

Comment:
No need for this variable. If `bytesToCopy > availableBytes` we return. In all other cases `sampleCount` is equal `bytesToCopy`, which you can use below.